tests/commands/get_object_lock_configuration.sh

```diff
@@ -6,11 +6,13 @@ get_object_lock_configuration() {
     log 2 "'get object lock configuration' command missing bucket name"
     return 1
   fi
-  lock_config=$(aws --no-verify-ssl s3api get-object-lock-configuration --bucket "$1") || local get_result=$?
-  if [[ $get_result -ne 0 ]]; then
+  if ! lock_config=$(aws --no-verify-ssl s3api get-object-lock-configuration --bucket "$1" 2>&1); then
     log 2 "error obtaining lock config: $lock_config"
+    get_object_lock_config_err=$lock_config
+    export get_object_lock_config_err
     return 1
   fi
+  lock_config=$(echo "$lock_config" | grep -v "InsecureRequestWarning")
   export lock_config
```

Comment:
I don't believe that these export calls are necessary - variable creation is global by default and should be available after the function is called.